test/integration/patients/worklist/reduced-schedule.js

```diff
@@ -49,16 +47,25 @@ context('reduced schedule page', function() {
 
         fx.data = fx.data.slice(0, 3);
 
+        fx.included.push({
+          id: '1',
+          type: 'flows',
+          attributes: {
+            name: 'Complex Care Management',
+            details: null,
+            created_at: testTs(),
+            updated_at: testTs(),
+          },
+        });
+
         fx.included.push(
           {
             id: '1',
             type: 'patients',
-            attributes: _.extend({}, _.sample(fxPatients), { first_name: 'Test', last_name: 'Patient' }),
-          },
-          {
-            id: '1',
-            type: 'flows',
-            attributes: _.extend({}, _.sample(fxFlows), { name: 'Complex Care Management' }),
+            attributes: {
+              first_name: 'Test',
+              last_name: 'Patient',
+            },
           },
         );
```

Comment:
What if we just don't import the fixture data. And just manually create the patient/flow fixture instead? Then we don't have to worry about any mutating issues.

Doesn't really seem necessary to import the entire fixture json data in these instances.

Reply:
That's probably ok. The benefit of the fixtures data is that we can update the base fixtures and anywhere they're used in the app will also update.. so if in the future we add something to some model, and if all instances of that model across the code base need that new data not to break, then fixture data where we're modifying a couple of items will work, but the incomplete fixtures we're storing in the test will not.

Reply:
Hmm ya I didn't think of that...